roles/properties/main/tasks/restore.yml

```diff
@@ -26,10 +29,10 @@
   loop: "{{ tar_files.files | map(attribute='path') | list }}"
 
 - name: Restore MySQL database from the dump
-  shell: "mysql -u nobody {{ db_name }} < {{ item | regex_replace('(.tar.bz2)$', '')}}/{{ db_name }}.sql"
-  loop: "{{ tar_files.files | map(attribute='path') | list }}"
+  shell: "mysql -u nobody {{ db_name }} < {{ restore_dir }}/{{ backup_name }}/{{ db_name }}.sql"
```

Comment:
`backup_name` might not match with the latest `backup_name` no? thats why i changed and added the looped item with the regex to replace the `.tar.bz2` extension. and it was working last time i checked..🤞